src/itest/kotlin/nl/lifely/zac/itest/client/KeycloakClient.kt

```diff
@@ -28,11 +28,11 @@ object KeycloakClient {
             .url("$KEYCLOAK_HOSTNAME_URL/realms/$KEYCLOAK_REALM/protocol/openid-connect/token")
             .post(
                 FormBody.Builder()
-                    .add("client_id", KEYCLOAK_CLIENT)
+                    .add("client_id", ZAC_KEYCLOAK_CLIENT)
```

Comment:
To be honest, renaming this causes a bit of noise within the actual logic changes. Maybe better to leave this out for now?

Reply:
It has to be a warning indeed

Reply:
Reverted to the old names